### PR TITLE
Note on the folder structure for annotations

### PR DESCRIPTION
**Pull request by zonky2**
Note on the folder structure for annotations
See https://contao.slack.com/archives/CK4J0KNDB/p1635147545044300

### CODE REVIEW

File: docs/dev/framework/hooks.md [hunk 1]
```diff
@@ -131,6 +131,8 @@ legacy registered hooks.
 {{< version "4.8" >}}
 
 Contao also supports its own annotation formats via the [Service Annotation Bundle][ServiceAnnotationBundle].
+This means that it is only necessary to create one file for the respective adaptation. It should be noted,
+however, that one adheres to the provided [Contao folder structure][ContaoFolderStructure].
 Hooks can be registered using the `@Hook` Annotation. See the following example:
```
Comment (fiedsch) [inline docs/dev/framework/hooks.md:136]: has to adhere to

Comment (zonky2) [reply docs/dev/framework/hooks.md:136]: @fiedsch add diff in your comment pls

Comment (fiedsch) [reply docs/dev/framework/hooks.md:136]: will do so later, when I’m on my computer. I‘m using my cellphone atm.

File: docs/dev/framework/hooks.md [hunk 2]
```diff
@@ -131,6 +131,8 @@ legacy registered hooks.
 {{< version "4.8" >}}
 
 Contao also supports its own annotation formats via the [Service Annotation Bundle][ServiceAnnotationBundle].
+This means that it is only necessary to create one file for the respective adaptation. It should be noted,
+however, that one adheres to the provided [Contao folder structure][ContaoFolderStructure].
```
Comment (fiedsch) [inline docs/dev/framework/hooks.md]:
```suggestion
however, that one has to adhere to the provided [Contao folder structure][ContaoFolderStructure].
```

File: docs/dev/framework/hooks.md (hunk 1 above)
Comment (fiedsch) [reply docs/dev/framework/hooks.md:136]: Here you go (hope I did the diff proposal correctly).

File: docs/dev/framework/hooks.md [hunk 3]
```diff
@@ -131,6 +131,8 @@ legacy registered hooks.
 {{< version "4.8" >}}
 
 Contao also supports its own annotation formats via the [Service Annotation Bundle][ServiceAnnotationBundle].
+This means that it is only necessary to create one file for the respective adaptation. It should be noted,
+however, that one has to adhere to the provided [Contao folder structure][ContaoFolderStructure].
```
Comment (fritzmg) [inline docs/dev/framework/hooks.md]: @zonky2 this applies to everything, not just hooks. Also using DCA files in `src/Resources/contao`, like you did, should not cause any troubles with the service auto registration of the Contao Managed Edition. Something else must have been the issue.

Comment (zonky2) [reply docs/dev/framework/hooks.md]: DCA files in `src/Resources/contao` works fine!

**Comment by fritzmg** [comment]
The original issue has been solved in the mean time. But merging this with some adjustments anyway. Thank you @zonky2

**Pull request merged**